src/hooks/api/cities.ts

```diff
@@ -0,0 +1,69 @@
+import levenshtein from 'fast-levenshtein';
+import { QueryFunctionContext, useQuery } from 'react-query';
+
+import citiesBE from '../../../public/assets/citiesBE.json';
+import citiesNL from '../../../public/assets/citiesNl.json';
+
+type City = {
+  label: string;
+  name: string;
+  zip: string;
+};
+
+type GetCitiesArguments = {
+  q: string;
+  country: 'BE' | 'NL';
```

Comment:
Make a seperate type for country or is it overkill?

Reply:
I've added a constant and type now ✅